<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="5">
     <widget class="QSpinBox" name="font_size_button">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="default_button">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px; background-color: white;border: 1px solid grey;</string>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="bold_button">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px; background-color: white;border: 1px solid grey;</string>
      </property>
      <property name="text">
       <string>Bold</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="italic_button">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px; background-color: white;border: 1px solid grey;</string>
      </property>
      <property name="text">
       <string>Italic</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="underline_button">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px; background-color: white;border: 1px solid grey;</string>
      </property>
      <property name="text">
       <string>Underline</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="8">
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PyWord</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="7">
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionDefault"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSpelling"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default </string>
   </property>
  </action>
  <action name="actionSpelling">
   <property name="text">
    <string>Spelling</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
